app/src/main/java/ar/scacchipa/twittercloneapp/fragment/FragmentAuthWebDialog.kt

```diff
@@ -33,10 +34,11 @@ class FragmentAuthWebDialog : Fragment() {
             when (token) {
 
                 is ResponseDomain.Success<*> -> {
-                    val action = FragmentAuthWebDialogDirections
-                        .actionFragmentAuthWebDialogToFragmentHome(
-                            (token.data as UserAccessTokenDomain).accessToken)
-                    findNavController().navigate(action)
+                    startActivity(
+                        Intent(
+                            activity,
+                            MainCloneActivity::class.java
+                        ))
```

Comment:
I think here you are missing a step. There is a bug here. See the video attached
[errorBackStack.webm](https://user-images.githubusercontent.com/101664313/184703749-582ad220-4825-4f97-8ebd-285fcce268af.webm)
.

Reply:
The following code was added.

val intent = Intent(  ...., .... )
intent.flags =  Intent.FLAG_ACTIVITY_NEW_TASK or
                            Intent.FLAG_ACTIVITY_CLEAR_TASK

(these flags clear the entire the history stack when a new activity is started.